src/main/webapp/features/shared/services/game/GameModel.service.js

```diff
@@ -0,0 +1,95 @@
+                    status = STATUS_LOADED;
+                }
+
+                return {
+                    isLoaded: function() {
+                        return status === STATUS_LOADED;
+                    },
+                    load: function() {
+                        var deferred = $q.defer();
+
+                        Remote.game.details({gameId: id}).then(function(response) {
+                            details = response.data;
+                            status = STATUS_LOADED;
+                            deferred.resolve();
+                        }, function() {
+                            deferred.reject();
+                        });
+
+                        return deferred.promise;
+                    },
+                    startUpdating: function(delay, onEverySuccess, onEveryError) {
+                        var self = this;
+                        this.stopUpdating();
+
+                        (function runTimeout() {
+                            updatingTimeout = $timeout(function() {
+                                self.load().then(function() {
+                                    if (onEverySuccess) {
+                                        onEverySuccess();
+                                    }
+                                    runTimeout();
+                                }, function() {
+                                    if (onEveryError) {
+                                        onEveryError();
+                                    }
+                                    runTimeout();
+                                });
+                            }, delay);
+                        })();
+                    },
+                    stopUpdating: function() {
+                        $timeout.cancel(updatingTimeout);
+                    },
+                    get: function() {
+                        return details;
+                    },
+                    getId: function() {
+                        return id;
+                    },
+                    isNew: function() {
+                        return details.status === "NEW";
+                    },
+                    isPlaying: function() {
+                        return details.status === "PLAYING";
+                    },
+                    isCurrentUserCreator: function() {
+                        return details.creatorId === User.get().id;
+                    },
+                    getCurrentUser: function() {
+                        for (var i in details.gameUsers) {
+                            if (details.gameUsers[i].user.id === User.get().id) {
+                                return details.gameUsers[i];
+                            }
+                        }
+                        return false;
+                    },
+                    isCurrentUserReady: function() {
+                        var user = this.getCurrentUser();
+                        return (user) ? user.ready : undefined;
```

Comment:
just return 'false', no? you never check that this method return 'undefined', you check only that it can be true or false, if method returns 'undefined', it will be converted to 'false' automatically, so I can't see any reason to return undefined, maybe because in Java we always return only true or false in methods that related to boolean logic.